boards/ite/it82xx2_evb/Kconfig.defconfig

```diff
@@ -3,9 +3,6 @@
 
 if BOARD_IT82XX2_EVB
```

Comment:
Needs an index.rst in the ite folder, copy one of the other ones and update vendor name and title

Reply:
done